### ISSUE TITLE
RBG parsing reports error when var function used

### ISSUE THREAD

**Issue by volkanceylan** [trullock/NUglify#287]
For a CSS definition like this:

```
a {
    --rgb-nums: 110 120 130;
    color: rgb(var(--rgb-nums));
}
```

Nuglify reports a syntax error. The produced output is fine though.

It's possible to use `rgba(var(--rgb-nums) / 100%)` to avoid the error but if someone else used this syntax it fails Bundler Minifier tool etc. with its default settings.

I'm going to open a pull request with a fix, so creating this issue just for reference.

Thank you for this lovely project and your hard work. As a mainly .NET developer I like avoiding Node JS ecosystem where possible (not so feasible nowadays though), and this tool generally runs faster according to my testing.

**Comment by trullock** [comment]
Merged your first PR, send me the second to fix this :)

**Comment by trullock** [comment]
Fixed in 1.17.10
